NtupleProducer/plugins/HTauTauNtuplizer.cc

```diff
@@ -293,6 +297,14 @@ class HTauTauNtuplizer : public edm::EDAnalyzer {
   Float_t _PuppiMETCov10;
   Float_t _PuppiMETCov11;
   Float_t _PuppiMETsignif;
```

Comment:
Just a very naive question: there's no "MET significance" equivalent for DeepMET?

Reply:
Not as far as I know

Reply:
thanks, this is not a critical issue I suppose, we might just need to check the implications with SVFit which uses the covariance matrix if we decide to switch to deepMET for the whole analysis